### PR TITLE
Allow claiming rewards

### PR DESCRIPTION
**Pull request by lcamargof**

### CODE REVIEW

**Review by JuampiRombola — APPROVED** [review]

**Review by JuampiRombola — APPROVED** [review]
LGTM. Just consider to use `React.memo` in components with props to skip unnecessary re-rendering.

File: src/views/auctions/auctions-sidebar/claim-rewards/components/ClaimFromTraderButton.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const transaction = useMemo(() => {
+    if (!erc20s.length || !rTokenContracts) {
+      return undefined
+    }
+
+    const collateralsToClaim: Set<Address> = new Set()
+
+    for (const reward of erc20s) {
+      reward.collaterals.forEach((c) => collateralsToClaim.add(c))
+    }
```
Comment (JuampiRombola) [inline src/views/auctions/auctions-sidebar/claim-rewards/components/ClaimFromTraderButton.tsx:47]:
```suggestion
    erc20s.flatMap(({ collaterals }) => collaterals).forEach((c) => collateralsToClaim.add(c))
```

**Review by JuampiRombola — APPROVED** [review]
lgtm

**Review by pmckelvy1 — APPROVED** [review]
lgtm

**Pull request merged**